src/app/options/qgsoptions.cpp

```diff
@@ -1613,6 +1614,8 @@ void QgsOptions::saveOptions()
 
   mSettings->setValue( QStringLiteral( "/qgis/checkVersion" ), cbxCheckVersion->isChecked() );
   mSettings->setValue( QStringLiteral( "/qgis/dockAttributeTable" ), cbxAttributeTableDocked->isChecked() );
+  mSettings->setValue( QStringLiteral( "/qgis/autosizeAttributeTable" ), cbxAutosizeAttributeTable->isChecked() );
```

Comment:
could you use the new settings API to add new settings please?
see https://docs.qgis.org/3.34/en/docs/developers_guide/codingstandards.html#settings

Reply:
No worries, will do. I will update this PR with the requested changes hopefully in the next few days.